spec/osut_tests_spec.rb

```diff
@@ -2708,8 +2712,9 @@ module M
     expect(core.floorArea).to be_within(TOL).of(149.66)
     core_volume = core.floorArea * 3.05
     expect(core_volume).to be_within(TOL).of(core.volume)
-    expect(attic.volume).to be_within(TOL).of(720.19)
-    expect(attic.floorArea).to be_within(TOL).of(567.98)
+    expect(attic.volume).to be_within(TOL).of(720.19) unless v < 350
```

Comment:
Calculations of core and attic floor/ceiling surface areas (m2), as well as core and attic volumes (m3), remain consistent even with 'mini' surfaces inserted in the centre of each. This however requires sequencing vertices of the larger (original) surfaces, as described here and here. Simulation results remain consistent, regardless of Solar Distribution modes, as shown here.

Reply:
You should check the surface areas and zone volumes in the E+ results too. The reporting measure has an example here: https://github.com/NREL/openstudio-common-measures-gem/blob/develop/lib/measures/example_report/resources/os_lib_reporting_example.rb#L221 that shows how to query data out of the E+ tabular results.  The eplusout.sql file has a table named `tabulardatawithstrings` which follows the tables in the eplustbl.htm file.  If you adjust the `ReportName`, `ReportForString`, `TableName`, `RowName`, `ColumnName`, and `Units` in the query you can get any of the other data in the eplustbl.htm file like the "Envelope Summary" or "Zone Summary"

Reply:
Thanks for the pointer, @macumber. Reported areas/volumes in the _eplusout.htm_ do match these first 3x test results. Once I'm done, I'll look into automating an SQL test.